.env

```diff
@@ -3,7 +3,7 @@ PORT=3000
 
 #db credential
 DB_USER_NAME=postgres
-DB_PASSWORD=kaizen
+DB_PASSWORD=3232
```

Comment:
never commit env file changes . please revert it

Reply:
Ok